internal/sdkv2provider/resource_cloudflare_access_keys_configuration.go

```diff
@@ -92,6 +92,9 @@ func resourceCloudflareKeysConfigurationImport(ctx context.Context, d *schema.Re
 	d.SetId(accountID)
 	d.Set(consts.AccountIDSchemaKey, accountID)
 
-	resourceCloudflareAccessKeysConfigurationRead(ctx, d, meta)
+	if err := resourceCloudflareAccessKeysConfigurationRead(ctx, d, meta); err != nil {
+		return nil, fmt.Errorf("diagnostic error occurred: %v", err)
```

Comment:
since this will be dumping a `diag.Diagnostics` struct, this is going to be pretty unreadable.

```
[{0 My Summary Some more details here []}]
``` 
instead we probably want to pull out the first (and likely only `diag.Diagnostic`) to return that as the error.
```suggestion
		return nil, errors.New(err[0].Summary)
```